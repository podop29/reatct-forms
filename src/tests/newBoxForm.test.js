import React from "react"
import {render} from "@testing-library/react"
import NewBoxFrom from "../newBoxForm"

it("renders witout crashing", ()=>{
    render(<NewBoxFrom/>)
})

it("matches snapshot", ()=>{
    const {asFragment} = render(<NewBoxFrom/>);
    expect(asFragment()).toMatchSnapshot()
})