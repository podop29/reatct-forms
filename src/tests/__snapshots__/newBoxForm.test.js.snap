// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <form>
      <input
        name="width"
        placeholder="Width"
        type="text"
        value=""
      />
      <input
        name="height"
        placeholder="Height"
        type="text"
        value=""
      />
      <input
        name="backgroundColor"
        placeholder="background Color"
        type="text"
        value=""
      />
      <button>
        Make Box
      </button>
    </form>
  </div>
</DocumentFragment>
`;
